pkg/nvml/types_gen.go

```diff
@@ -1,345 +0,0 @@
-// Code generated by cmd/cgo -godefs; DO NOT EDIT.
```

Comment:
This file should definitely not be deleted....

Reply:
Seems my docker vm has a full disk.